source/documentation/runbooks/004-analytical-platform-compute-maintenance.html.md.erb

```diff
@@ -0,0 +1,117 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Analytical Platform Compute Maintenance - Draft
```

Comment:
```suggestion
title: Analytical Platform Compute Maintenance
```

Reply:
I added the "draft" to acknowledge the document was still a work in progress until after a run through of the document to complete the patching - can remove or leave @Gary-H9

Reply:
removed